composer.json

```diff
@@ -4,14 +4,14 @@
     "type": "sulu-bundle",
     "license": "MIT",
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.2 || ^8.3",
```

Comment:
Should also match the sulu 2.5 requirements ^8.0/^8.1

Therefore also the CI should run for these php versions